src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+    text-decoration: none;
+    min-height: 200px;
+    overflow: auto;
+    display: grid;
+    margin-top: 66px;
+    opacity: 1;
+}
+
+.rectangle {
+    grid-row: 1;
+    grid-column: 1;
+    opacity: 0.3;
+    background-color: #2A2C2F;
+    z-index: 1;
```

Comment:
Usully you do not need `z-index` without `position: absolute`

Reply:
z-index helps here so the title is on top of the rectangle that is on top of the picture. Without z-index it breaks.

Reply:
It does not. If you remove this line nothing will change.
![image](https://user-images.githubusercontent.com/42805085/212661089-f2c39bc1-9700-4f48-a099-d6d0d8ecf66b.png)